docs/vectors/index.rst

```diff
@@ -0,0 +1,44 @@
+Reference vectors
+=================
+
+In addition to the policy of keeping any code path in PyNaCl covered by
+unit tests, the output from cryptographic primitives and constructions
+must be verified as really being conformant to the reference implementations
+or standards.
+
+Imported reference vectors
+--------------------------
+
+Wherever possible, PyNaCl policy is to pull already-available reference
+vectors for primitive or constructions, in the original format, or applying
+the minimal changes needed for straightforward parsing.
+
+Box construction
+^^^^^^^^^^^^^^^^
+
+The reference vector for testing the :py:class:`nacl.public.Box`
+implementation come from libsodium's test/default/box.c and
```

Comment:
Let's wrap all paths with backticks (e.g. ``test/default/box.c``). GitHub's markdown is going to eat those, but rst needs double backtick.